include/mata/nfa.hh

```diff
@@ -172,7 +185,19 @@ struct Move {
         }
     }
 
+    // THIS BREAKS THE SORTEDNESS INVARIANT,
+    // dangerous,
+    // but useful for adding states in a random order to sort later (supposedly more efficient than inserting in a random order)
+    void inline push_back(const State s)
+    {
+        targets.push_back(s);
+    }
```

Comment:
How do you fix the sortedness? Do you call some method of ordered vector? If so, it should be added to the comment.

Reply:
No, we do not have any method to sort the underlying structure again. We should implement one, add mention to that function in here and create a similar comment to the sort function explaining that unless you explicitly call push_back with values that are unsorted, you have to sort the underlying structure again.

Reply:
we have sort_and_rmdupl (used in some version of trim), I will add that comment